deploy.sh

```diff
@@ -32,7 +32,7 @@ echo -e "\tAuditEventNamePrefix SSM key ${GREEN}$audit_event_name_prefix${NOCOLO
 echo -e "\tCriIdentifier SSM key ${GREEN}$cri_identifier${NOCOLOR}"
 
 ./gradlew clean
-sam validate -t infrastructure/lambda/template.yaml --config-env dev --lint
+sam validate -t infrastructure/lambda/template.yaml --config-env dev
```

Comment:
@Suad-Mohamed is there any way to keep the linting and resolve the error it raises?